extensions.json

```
@@ -712,6 +712,10 @@
       "download": "https://github.com/lgalabru/clarity-lsp/releases/download/v0.5.1/clarity-lsp.vsix",
       "version": "0.5.1"
     },
+    {
+      "id": "Hirse.vscode-ungit",
+      "download": "https://github.com/Hirse/vscode-ungit/releases/download/v2.2.3/vscode-ungit-2.2.3.vsix",
```

With `"download"` as with `"checkout"`, it's better to add an appropriate `"version"`, otherwise the script will always try to re-publish the same version every night. (It's an optimization.)

```suggestion
      "download": "https://github.com/Hirse/vscode-ungit/releases/download/v2.2.3/vscode-ungit-2.2.3.vsix",
      "version": "2.2.3"
```